src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -79,6 +79,13 @@
   public static final String LOGS_DIR_DEFAULT = "logs";
   public static final String LOGS_DIR_DISPLAY = "Logs directory";
 
+  public static final String AVRO_CODEC_CONFIG = "avro.codec";
+  public static final String AVRO_CODEC_DEFAULT = "null";
+  public static final String AVRO_CODEC_DISPLAY = "Avro Compression Codec";
+  public static final String AVRO_CODEC_DOC = "The Avro compression codec to be used for output  "
+      + "files. Available values: null, deflate, snappy and bzip2 (CodecSource is org.apache"
+      + ".avro.file.CodecFactory)";
```

Comment:
Should we provide a validator for this config?

Reply:
Added a validator here.